paywall/src/unlock.js/MainWindowHandler.ts

```diff
@@ -65,6 +77,26 @@ export default class MainWindowHandler {
       }
     })
 
+    // When the data iframe sends updates, store them in the mirror
+    this.iframes.data.on(PostMessages.UPDATE_LOCKS, locks => {
+      this.blockchainData.locks = locks
+    })
+    this.iframes.data.on(PostMessages.UPDATE_ACCOUNT, address => {
+      this.blockchainData.account = address
+    })
+    this.iframes.data.on(PostMessages.UPDATE_ACCOUNT_BALANCE, balance => {
```

Comment:
Do we care about anything outside of address currently/for this use case?

Reply:
The immediate need is just for the address, but all the data is being emitted at present anyway. I can see a concern that it might commit us to the current structure if we show it all. Would you prefer if we only put out an object with the account address in it for now?